app.css

```diff
@@ -69,4 +72,166 @@ body {
 .personal-social-info h3 {
   color: #7f8cff;
   font-size: 16px;
-}
\ No newline at end of file
+}
+
```

Comment:
Great job fixing the issues. But when you give a `font-family` for the `body` tag there is a typo error that's why the font is not looking the same as the Figma design. Please make sure to remove 'y' at the end of the `sans-serify` to its `font-family: 'Poppins', sans-serif;`. Then you're good to go.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/34578631/159118383-dbaf88ea-ee6e-4877-82c3-b5def9d67345.png">

Reply:
@Mre55 this is really helpful. thank you